content/working-groups/learning/minutes/2019.10.01-meeting.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Learning WG 2019-10-01 sync meeting
+type: docs
+---
+# Learning WG 2019-10-01 sync meeting
+
+- Team discussed that watching same video at the same time works well so far
```

Comment:
Sorry, my english is not great but ... wouldn't be better to say something like ... `Team discussed that having all of us watching the same video at the same time works well so for` like putting more emphasis that we indeed watched all the same video and discussed it.

Reply:
"at the same time" also implies that we watched it together

Reply:
But we didn't watched together at the same time. Still I just raised a doubt on the way to explain it and my english is ungreat so I'd leave it up to you. But I'm fine leaving it as is.

Reply:
You are correct about this wording. I felt there was something off. Let me edit this and re-push.

Reply:
Your English is good, @spastorino (it is my second language too)! Its just that my job was sloppy. :stuck_out_tongue:

Reply:
How about this 

```Team discussed that watching same video by all members, one video at a time, works well for everyone so far``` ?

Reply:
Seems good to me.